package.json

```diff
@@ -7,6 +7,7 @@
     "axios": "^0.21.1",
     "body-parser": "^1.18.2",
     "bunyan": "^1.8.15",
+    "classnames": "^2.3.1",
```

Comment:
classnames is super useful. don't leave home without it!

Reply:
A nice touch that will help modernize the codebase a bit :)